.github/workflows/pr-review-requester.yml

```diff
@@ -0,0 +1,19 @@
+name: pr-review-requester
+
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened, ready_for_review]
+
+jobs:
+    auto-add-reviewer:
+        name: Auto add reviewer to PR
+        if: github.event.pull_request
```

Comment:
if you remove the issue trigger, you can remove this.

Reply:
e709df605e8bb6b6bc97a12a1bbbdfa5f8e6ff0a and done, thanks again for the review